locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -1729,6 +1876,27 @@ protected JcloudsSshMachineLocation createJcloudsSshMachineLocation(ComputeServi
         }
     }
 
+    protected WinRmMachineLocation registerWinRmMachineLocation(String vmHostname, ConfigBag setup, String nodeId) {
+        WinRmMachineLocation winRmMachineLocation = createWinRmMachineLocation(vmHostname, setup);
+        winRmMachineLocation.setParent(this);
```

Comment:
I wonder why we don't set the parent inside `registerJcloudsSshMachineLocation`. Ignore this comment for this PR.